.github/workflows/rust.yml

```diff
@@ -158,3 +140,34 @@ jobs:
         run: wget -qO- https://github.com/thedodd/trunk/releases/latest/download/trunk-x86_64-unknown-linux-gnu.tar.gz | tar -xzf-
       - name: Build
         run: cd demo && ../trunk build
+
+  # ---------------------------------------------------------------------------
+
+  tests:
+    name: Run tests
+    # We run the tests on macOS because it will run with an actual GPU
```

Comment:
```suggestion
    # We run the tests on macOS because it will run with an actual GPU,
    # which is needed by the egui_kittest snapshot tests.
```